__snapshots__/side-drawer.md

```diff
@@ -3,7 +3,7 @@
 #### `should internal contents`
 
 ```html
-<aside class="side-drawer">
+<aside class="side-drawer side-drawer-dismissible">
```

Comment:
if it's dismissible by default why do we need its class?
for style appliance just do something like
```css
.side-drawer:not(.side-drawer-modal) {
  /* dismissible content here */
}
```